gen-compdocs/generators/doc.go

```diff
@@ -36,6 +36,17 @@ func (s byName) Len() int           { return len(s) }
 func (s byName) Swap(i, j int)      { s[i], s[j] = s[j], s[i] }
 func (s byName) Less(i, j int) bool { return s[i].Name() < s[j].Name() }
 
+const generated_warning = `<!--
+The file is auto-generated from the component source code. Please check
```

Comment:
Hi @tengqm . Here is some feedback. Reword as you like.

This file is auto-generated from the [component reference source code](https://github.com/kubernetes-sigs/reference-docs/gen-compdocs).  To learn how to generate the reference documentation, read [Contributing to the reference documentation](/docs/contribute/generate-ref-docs/). To update the reference content, follow the [Contribute upstream](/docs/contribute/generate-ref-docs/contribute-upstream/) guide. You can file document formatting bugs against the [reference-docs](https://github.com/kubernetes-sigs/reference-docs/) project.

Reply:
Will take your suggestion.